README.md

```diff
@@ -18,12 +18,12 @@ It relies on a few libraries:
 * [Boost](http://www.boost.org/) 1.40 or later.
   Required libraries: *thread*, *system*, *regex*, and *date_time*.
   Optional library for Boost Test driver: *test*.
-* [GTest](http://code.google.com/p/googletest/) 1.4 or later.
-  Optional for the GTest driver.
+* [GTest](http://code.google.com/p/googletest/) 1.6 or later.
```

Comment:
As you said, it should work with older releases (1.4) but really 1.6 has been released in 2011 so let's go with that! Perhaps we can add "It might work, but it is not guaranteed, to work with earlier versions of those libraries", or something on those lines?